File: WeLearnApp/src/main/java/com/welearn/WeLearnApp/dto/request/learningsession/LearningSessionCreationRequest.java
package com.welearn.WeLearnApp.dto.request.learningsession;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class LearningSessionCreationRequest {
    LocalDateTime startTime;
    long duration;
    int grade;
    String subject;
    String learningMethod;
    double tuition;
}

File: WeLearnApp/desktop-app/TutorApp/Models/ForAPI/Request/LearningSessionCreationRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TutorApp.Models.ForAPI.Request
{
    public class LearningSessionCreationRequest
    {
        public string Id { get; set; }
        public DateTime StartTime { get; set; }
        public long Duration { get; set; }
        public int Grade { get; set; }
        public string Subject { get; set; }
        public string LearningMethod { get; set; }
        public decimal Tuition { get; set; }

        public override string ToString()
        {
            return $"ID: {Id}, StartTime: {StartTime}, Duration: {Duration}, Grade: {Grade}, Subject: {Subject}, LearningMethod: {LearningMethod}, Tuition: {Tuition}";
        }
    }
}
